.github/workflows/tests.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: |
           mkdir bottles
           cd bottles
-          brew test-bot
+          brew test-bot --skip-recursive-dependents
```

Comment:
Is this PR mergable with this change? I know without it, testing OpenSSL tests an extremely long time.

Reply:
It is mergable but this commit will not be included in that merge. I put it in a separate commit for that reason.

Reply:
In the future we might have something better - but this is what we thought of over the weekend in anticipation of the OpenSSL update today.

Reply:
Thanks -- we (pyca/cryptography) redistribute a statically linked copy of OpenSSL from brew, so I'm very appreciative that y'all were planning ahead for today!